src/conversion_helper.js

```diff
@@ -580,7 +580,36 @@ function preprocessComment(comment) {
     return input;
   }
 
+  function abbreviationProcessor(comment) {
+    const abbreviations = {
+      k: 1000,
+      thousand: 1000,
+      m: 1000000,
+      mill: 1000000,
+    };
```

Comment:
Ohhh, this object is an excellent idea to make it easy to add new abbreviations!

One though I had is maybe we can use an array of regexes for the keys, like:
```
{
  [\k\, \thou(?:sand)\] : 1000,
  [\m\, \mill?\, \million\] : 1000000
}
```
And then we can use regex_helper.regexJoinToString to stitch the array together.

There's a tradeoff though, this would make the key a bit more unclear, but it would require less duplication with the numbers. What are your thoughts?

Reply:
I really like this idea since it enables us to have multiple keys matching to the same value, but If i'm not wrong json keys can only be strings, maybe instead of having an array with the regexes we could have a string with the regexes separated by a comma? (that wouldn't be as clean as having an array though)

Reply:
Oh, TIL lol. Two ideas:

1) We can do rh.regexJoinToString([\k\, \thou(?:sand)\]) and make it a string
2) Change the structure to look something like 
```
[
  { "regex" : [\k\, \thou(?:sand)\],  "value" : 1000 },
  { "regex" : [\m\, \mill?\, \million\] , "value" : 1000000 }
]
```

I like solution 2 better, because it might get annoying to do regexJoinToString so many times

Reply:
I like solution 2, I'll implement that.